stempy/image.cpp

```diff
@@ -0,0 +1,67 @@
+#include "image.h"
+#include "mask.h"
+
+#include <memory>
+
+using namespace std;
```

Comment:
We should try to get in the habit of just bringing in the symbols we use instead of the whole std namespace.

Reply:
Sure, I can do that.